packages/redis/src/index.ts

```diff
@@ -28,20 +33,28 @@ export interface IRedisOptions {
 }
 
 export type IRedisInputStreamOptions = IRedisOptions & {
-    readStreams: string[];
+    readonly readStream: string;
+    readonly consumerGroup: string;
+    readonly consumerId?: string;
+    readonly consumerGroupStartId?: string;
+    readonly blockTimeoutMs: number;
```

Comment:
should we make these three optional as well and provide sane defaults? `blockTimeout` could be `100ms`, idleTimeout `30s`

Reply:
Is 10 a sane default for `batchSize`?

Reply:
i think so